src/PlayAudioUsingMediaStreams/Controllers/AnimalSoundboardController.cs

```diff
@@ -33,93 +40,108 @@ public async Task Get()
-                                    if (soundToPlay == null)
-                                    {
-                                        continue;
-                                    }
-                                    Console.WriteLine($"Animal detected: {soundToPlay.Name}");
-                                    
-                                    var mediaMessage = new
-                                    {
-                                        streamSid, 
-                                        @event = "media", 
-                                        media = new
-                                        {
-                                            payload = soundToPlay.AudioDataAsBase64
-                                        }
-                                    };
-                                    
-                                    var rawJson = JsonSerializer.Serialize(mediaMessage);
-                                    var responseBuffer = Encoding.UTF8.GetBytes(rawJson);
-                                    
-                                    await webSocket.SendAsync(
-                                        new ArraySegment<byte>(responseBuffer, 0, responseBuffer.Length),
-                                        receiveResult.MessageType,
-                                        receiveResult.EndOfMessage,
-                                        CancellationToken.None);
-                                }
-                            }
-                        });
-                        break;
-                    
-                    case "start":
-                        streamSid = jsonDocument.RootElement.GetProperty("streamSid").GetString();
-                        Console.WriteLine($"StreamId: {streamSid}");
-                        break;
-                    
-                    case "media":
-                        string payload = jsonDocument.RootElement.GetProperty("media").GetProperty("payload").GetString();
-                        await _speechRecognitionService.SendAudio(payload);
-                        break;
-                    
-                    case "stop":
-                        break;
+                    payload = soundToPlay.AudioDataAsBase64
                 }
-            }
-            
-            receiveResult = await webSocket.ReceiveAsync(
-                new ArraySegment<byte>(buffer), CancellationToken.None);
-        }
+            };
+
+            var rawJson = JsonSerializer.Serialize(mediaMessage);
+            var responseBuffer = Encoding.UTF8.GetBytes(rawJson);
 
-        await webSocket.CloseAsync(
-            receiveResult.CloseStatus.Value,
-            receiveResult.CloseStatusDescription,
-            CancellationToken.None);
+            await webSocket.SendAsync(
+                new ArraySegment<byte>(responseBuffer, 0, responseBuffer.Length),
+                WebSocketMessageType.Text,
+                true,
```

Comment:
This should always be `WebSocketMessageType.Text` and `true` I believe, so using this instead of copying it from the incoming websocket message.